src/nvim/ugrid.h

```diff
@@ -20,15 +20,27 @@ struct ugrid {
   UCell **cells;
 };
 
+static inline UCell *ugrid_get_cell(UGrid *grid, int row, int col)
```

Comment:
I don't like this API change at all. I would rather just add the missing check where it causes a crash as an `if` condition,  without complicating the other array accesses for no reason.